cwms-data-api/src/main/java/cwms/cda/data/dto/TimeSeries.java

```diff
@@ -56,6 +53,14 @@ public class TimeSeries extends CwmsDTOPaginated {
     @Schema(description = "The units of the time series data",required = true)
     String units;
 
+    @Schema(description = "The type of time series query")
+    String dateVersionType;
```

Comment:
Use the enum here too instead of the string.

Reply:
That schema description needs more information too

Reply:
both have been updated